src/mcts/utils.h

```diff
@@ -226,7 +268,31 @@ class MctsResult {
   }
 
   // assume already normalized
-  void sample() {
+  void sampleWithoutSmoothing() {
+    auto weight = [this](float pival) {
+      return pival;
+    };
+    float maxWeight = 0.0f;
```

Comment:
you should use `auto maxWeight = std::max_element(mctsPolicy.begin(), mctsPolicy.end()`

Reply:
https://en.cppreference.com/w/cpp/algorithm/max_element

Reply:
Thanks for this. Note that this was their code (it just looks like I wrote it because I was making a copy of their function). Is it better to fix my function and their function, or to leave them both as is?

Reply:
Fix them both! Should be pretty quick.